pkg/kube/client.go

```diff
@@ -56,7 +56,7 @@ import (
 )
 
 const (
-	// MissingGetHeader is added to Get's outout when a resource is not found.
+	// MissingGetHeader is added to get output when a resource is not found.
```

Comment:
The "Get's" is actually intentional as it's referring to the function `Get`. The typo fix for output is good though. Can you revert that change, please?

Reply:
OK!